gateway/handlers/reader.go

```diff
@@ -65,3 +78,36 @@ func MakeFunctionReader(metricsOptions metrics.MetricOptions, c *client.Client)
 		w.Write(functionBytes)
 	}
 }
+
+func getReplicaInfo(c *client.Client, services []swarm.Service, ctx context.Context) map[string]int {
+	// Begin replica info section
+	taskFilter := filters.NewArgs()
+
+	taskFilter.Add("desired-state", "running")
+	tasks, err := c.TaskList(ctx, types.TaskListOptions{Filters: taskFilter})
+	if err != nil {
+		fmt.Println(err)
+	}
+
+	nodes, err := c.NodeList(ctx, types.NodeListOptions{})
```

Comment:
Is this necessary?

Reply:
Right you are, this is unnecessary.

Reply:
Correction, I believe this may be necessary. I removed this code, then thought that I re-deployed the new image, but it turns out it was the old image. Now that I have this removed, I see that it shows every function as having replica status `1/1`, even though `docker service ls` may show the same function as `0/1`. I think filtering through the `NodeList` may be necessary

Reply:
Re-correction, I believe it's not necessary to loop through `nodes`.